apps/system/test/unit/update_manager_test.js

```diff
@@ -1244,6 +1244,20 @@ suite('system/UpdateManager', function() {
       });
     });
 
+    suite('cancel prompt continued', function() {
+      setup(function() {
+        UpdateManager._downloading = true;
+        UpdateManager._uncompressing = true;
```

Comment:
Please avoid setting internal variables as part of the setup, 
https://github.com/mikehenrty/gaia/blob/bug-1079244-fota-no-stop/apps/system/test/unit/update_manager_test.js#L603-605
should do the trick